src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Dropdown from "react-dropdown";
+import styles from "./Dropdown.scss";
+
+const options = ["szkola1", "szkola2", "szkola3"];
+
+class Drop extends React.Component {
```

Comment:
There's no reason to save keystrokes here. We can name it `DropdownComponent` as `Drop` is pretty unclear.